test/data/ansible/check_ostree.yaml

```diff
@@ -314,10 +294,10 @@
         - name: failed count + 1
           set_fact:
             failed_counter: "{{ failed_counter | int + 1 }}"
-      when: (edge_type == "none") and ((ansible_facts['distribution'] == 'Fedora' and ansible_facts['distribution_version'] is version('37', '<')) or
-            (ansible_facts['distribution'] == 'CentOS') or (ansible_facts['distribution'] == 'RedHat'))
+      when: sysroot_ro == "false"
 
-    - name: /sysroot should be mount with ro permission for edge-simplified-installer and edge-raw-image
+    # https://fedoraproject.org/wiki/Changes/Silverblue_Kinoite_readonly_sysroot
+    - name: /sysroot should be mount with ro permission on RHEL 9.2 , Centos9 and Fedora 37 above
```

Comment:
nitpick: since the task does not execute conditionally based on OS name or release, but only based on `sysroot_ro` variable, it IMO does not make sense to mention RHEL, CS and Fedora in its name at all. This difference is captured by the test case scripts that execute this playbook

Reply:
Yes, from the test script people can see the difference there. 
I will take care of it in next pr, as it took so loong to pass CI test.